lib/screens/profile/user_profile.dart

```diff
@@ -66,6 +68,16 @@ class ProfileDetailScreen extends StatelessWidget {
                   ],
                 ),
               ),
+              Center(
+                  child: StyledButton(
+                      child: const Text("Search users/Add friends"),
+                      onPressed: () {
+                        Navigator.push(
```

Comment:
Für die Weiterleitung warte vielleicht noch ab bis dieser PR #61 gemerged ist. Dort gibt es in dem Dialog ein Button, der auf diese Seite weiterleiten soll